logic/cacheLogic.js

```diff
@@ -52,7 +52,7 @@ module.exports = class CacheLogic {
   static async findContractEvents() {
     const fetchContractEvents = async () => {
       const contractTransactions = await aeternity.middlewareContractTransactions();
-      return contractTransactions.map(tx => tx.hash).asyncMap(aeternity.transactionEvents);
+      return contractTransactions.map(tx => tx.hash).asyncMap(hash => aeternity.transactionEvents(hash));
```

Comment:
can this be reverted to original state?

Reply:
The original state is broken, so we can either
a) merge your PR soon
b) leave as is